corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
I think it'd be better to find a way to reuse it. people who have been looking into js frameworks might have some advice, but one thing you can do is declare a variable for the `csrf` package that lives in the namespace of the page, and then you can reference that in other places (and you just assume that it's been loaded on any page that needs it). you can see an example of this pattern in how `dimagisphere` works in javascript [here](https://github.com/dimagi/commcare-hq/pull/9224/)